module/x/peggy/keeper/keeper.go

```diff
@@ -260,15 +308,22 @@ func (k Keeper) GetBatchConfirmByNonceAndTokenContract(ctx sdk.Context, nonce ui
 /////////////////////////////
 
 // SetEthAddress sets the ethereum address for a given validator
-func (k Keeper) SetEthAddress(ctx sdk.Context, validator sdk.ValAddress, ethAddr string) {
+func (k Keeper) SetEthAddress(ctx sdk.Context, validatorAddress sdk.ValAddress, ethAddress common.Address) {
```

Comment:
Are you moving to using `ethcommon.Address` throughout the codebase? Thats a possible refactor I don't hate, but provides questionable benefits.

Reply:
yeah it's the de facto address from ethereum and we should use it whenever possible 👍